layers/meta-resin-artik/recipes-kernel/linux/files/artik53x_add_dts_spi_node.patch

```diff
@@ -0,0 +1,39 @@
+From 797ceae96b837b8612ad41749d6398ba40c539b0 Mon Sep 17 00:00:00 2001
+From: Vicentiu Galanopulo <vicentiu@balena.io>
+Date: Mon, 26 Nov 2018 16:33:50 +0100
+Subject: [PATCH] eagleye_530s_add_dts_spi_node
+
+Signed-off-by: Vicentiu Galanopulo <vicentiu@resin.io>
+
+Upstream-Status: Inappropriate [configuration] 
+
+---
+ arch/arm/boot/dts/s5p4418-artik533-compy.dts | 12 ++++++++++++
+ 1 file changed, 12 insertions(+)
+
+diff --git a/arch/arm/boot/dts/s5p4418-artik533-compy.dts b/arch/arm/boot/dts/s5p4418-artik533-compy.dts
+index 2b342df..df06f68 100644
+--- a/arch/arm/boot/dts/s5p4418-artik533-compy.dts
++++ b/arch/arm/boot/dts/s5p4418-artik533-compy.dts
+@@ -504,6 +504,18 @@
+ 	pinctrl-0 = <&serial1_pin>, <&serial1_flow_cts>, <&serial1_flow_rts>;
+ };
+ 
++&spi2 {
```

Comment:
If one would want to modify the default .dtb like you did here, I think the node should actually be `spi_2`, because that is what this .dtsi expects: https://github.com/SamsungARTIK/linux-artik/blob/release/A533s-OS-18.05.00/arch/arm/boot/dts/s5p4418.dtsi#L48

And anyway, the proper way to enable spi on compy boards would be:

> # mkdir /sys/kernel/config/device-tree/overlays/spi2

then

# cat /mnt/boot/overlays/s5p4418-artik533-compy-spi2-overlay.dtbo > mkdir /sys/kernel/config/device-tree/overlays/spi2/dtbo